scaling_performance/using_cluster_loader.adoc

```diff
@@ -0,0 +1,259 @@
+
+|`total`
+|This field is not used.
+
+|`num`
+|An integer. The number  of pods or templates to deploy.
+
+|`image`
+|A string. The docker image URL to a repository where it can be pulled.
+
+|`basename`
+| A string. One definition of the base name for the template (or pod) that you want to create.
+
+|`file`
+|A string. The path to a local file, which is either a PodSpec or template to be created.
+
+|`parameters`
+|Key-value pairs. Under `parameters`, you can specify a list of values to
+override in the pod or template.
+|===
+
+.Fields under `tuningset`
+|===
+|Field |Description
+
+|`name`
+|A string. The name of the tuning set which will match the name specified when
+defining a tuning in a project.
+
+|`pods`
+|A sub-object identifying the `tuningset` that will apply to pods.
+
+|`templates`
+|A sub-object identifying the `tuningset` that will apply to templates.
+|===
+
+.Fields under `tuningset` `pods` or `tuningset` `templates`
+|===
+|Field |Description
+
+|`stepping`
+|A sub-object. A stepping configuration used if you want to create an object in a
+step creation pattern.
+
+|`rate_limit`
+|A sub-object. A rate-limiting tuning set configuration to limit the object
+creation rate.
+|===
+
+.Fields under `tuningset` `pods` or `tuningset` `templates`, `stepping`
+|===
+|Field |Description
+
+|`stepsize`
+|An integer. How many objects to create before pausing object creation.
+
+|`pause`
+|An integer. How many seconds to pause after creating the number of objects
+defined in `stepsize`.
+
+|`timeout`
+|An integer. How many seconds to wait before failure if the object creation is
+not successful.
+
+|`delay`
+|An integer. How many milliseconds (ms) to wait between creation requests
+|===
+
+[[configuring-cluster-loader-configuration-file-example]]
+=== Example Cluster Loader Configuration File
+
+Cluster Loader’s configuration file is a basic YAML file:
+
+----
+provider: local
+ClusterLoader:
+  cleanup: true
+  projects:
+    - num: 1
+      basename: clusterloader-cakephp-mysql
+      tuning: default
+      templates:
+        - num: 1
+          file: ./examples/quickstarts/cakephp-mysql.json
+
+    - num: 1
+      basename: clusterloader-dancer-mysql
+      tuning: default
+      templates:
+        - num: 1
+          file: ./examples/quickstarts/dancer-mysql.json
+
+    - num: 1
+      basename: clusterloader-django-postgresql
+      tuning: default
+      templates:
+        - num: 1
+          file: ./examples/quickstarts/django-postgresql.json
+
+    - num: 1
+      basename: clusterloader-nodejs-mongodb
+      tuning: default
+      templates:
+        - num: 1
+          file: ./examples/quickstarts/nodejs-mongodb.json
+
+    - num: 1
+      basename: clusterloader-rails-postgresql
+      tuning: default
+      templates:
+        - num: 1
+          file: ./examples/quickstarts/rails-postgresql.json
+
+  tuningset: <1>
+    - name: default
+      pods:
+        stepping: <2>
+          stepsize: 5
+          pause: 0 min
```

Comment:
@sjug I noticed the pause here is defined in minutes, while the description for `pause` describes the value in seconds. Should this be adjusted, or is `min` also valid?

Reply:
As far as I can see, min and ms should be removed.  Pause and timeout is in seconds, delay in rate_limit in milliseconds.  Good catch!

Reply:
Hmmm, this is another relic that I should probably fix in the OpenShift version. In the k8s Cluster Loader you can enter in time like this and it will be automatically converted to the correct time type. OpenShift on the other hand is hardcoded, which is... not ideal.